terratorch/cli_tools.py

```diff
@@ -101,6 +99,20 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     logging.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path):
+    if custom_modules_path:
+        if os.path.isdir(custom_modules_path):
+
+            # Add 'custom_modules' folder to sys.path
+            workdir, module_dir = os.path.split(custom_modules_path)
+
+            if not workdir:
```

Comment:
I think this line assumes that a path that does not have a `workdir` must therefore be in the current directory. For example, if `custom_modules_path = "/module"`. You would then erroneously add `.` when you should add `/`. I think if you look at the method in the Path class, you should be able to find something better than manually splitting the path.